java-checks/src/main/java/org/sonar/java/checks/CheckList.java

```diff
@@ -20,66 +20,23 @@
 package org.sonar.java.checks;
 
 import com.google.common.collect.ImmutableList;
-import org.sonar.java.checks.naming.BadAbstractClassNameCheck;
-import org.sonar.java.checks.naming.BadClassNameCheck;
-import org.sonar.java.checks.naming.BadConstantNameCheck;
-import org.sonar.java.checks.naming.BadFieldNameCheck;
-import org.sonar.java.checks.naming.BadFieldNameStaticNonFinalCheck;
-import org.sonar.java.checks.naming.BadInterfaceNameCheck;
-import org.sonar.java.checks.naming.BadLocalVariableNameCheck;
-import org.sonar.java.checks.naming.BadMethodNameCheck;
-import org.sonar.java.checks.naming.BadPackageNameCheck;
-import org.sonar.java.checks.naming.BadTestClassNameCheck;
-import org.sonar.java.checks.naming.BadTestMethodNameCheck;
-import org.sonar.java.checks.naming.BadTypeParameterNameCheck;
-import org.sonar.java.checks.naming.ClassNamedLikeExceptionCheck;
-import org.sonar.java.checks.naming.EnumAsIdentifierCheck;
-import org.sonar.java.checks.naming.FieldNameMatchingTypeNameCheck;
-import org.sonar.java.checks.naming.MethodNameSameAsClassCheck;
-import org.sonar.java.checks.naming.MethodNamedEqualsCheck;
-import org.sonar.java.checks.naming.MethodNamedHashcodeOrEqualCheck;
-import org.sonar.java.checks.serialization.CustomSerializationMethodCheck;
-import org.sonar.java.checks.serialization.ExternalizableClassConstructorCheck;
-import org.sonar.java.checks.serialization.PrivateReadResolveCheck;
-import org.sonar.java.checks.serialization.SerialVersionUidCheck;
-import org.sonar.java.checks.serialization.SerializableComparatorCheck;
-import org.sonar.java.checks.serialization.SerializableFieldInSerializableClassCheck;
-import org.sonar.java.checks.serialization.SerializableObjectInSessionCheck;
-import org.sonar.java.checks.serialization.SerializableSuperConstructorCheck;
-import org.sonar.java.checks.unused.UnusedLabelCheck;
-import org.sonar.java.checks.unused.UnusedLocalVariableCheck;
-import org.sonar.java.checks.unused.UnusedMethodParameterCheck;
-import org.sonar.java.checks.unused.UnusedPrivateFieldCheck;
-import org.sonar.java.checks.unused.UnusedPrivateMethodCheck;
-import org.sonar.java.checks.unused.UnusedReturnedDataCheck;
-import org.sonar.java.checks.unused.UnusedTestRuleCheck;
-import org.sonar.java.checks.unused.UnusedTypeParameterCheck;
+
+import java.util.List;
+
+import org.sonar.java.checks.naming.*;
+import org.sonar.java.checks.serialization.*;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Explicitly import the specific classes needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2208)